zeeguu/core/ml_models/paywall_detector.py

```diff
@@ -0,0 +1,8 @@
+
+from langdetect import detect
+from .utils import pre_process
+
+def is_paywalled(article_txt:str, model):
```

Comment:
this function feels funny. it is a very thin wrapper around `model.predict`. 
it almost feels like it's not needed. 
also, from the POV of the caller, they need to choose the model separately and pass it in.
why don't we select the model here, and let the caller, simply write: `paywalled = is_paywalled(text)`? 
then all the decisions related to the paywall detector are localized here.

Reply:
I agree with this, I made the model be loaded in this file and the prediction just using the globally defined model in the file.